Update hello-feature-pin-reset.md (#5554)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md

> To configure PIN reset on Windows devices you manage, use an [Intune Windows 10 custom device policy](https://docs.microsoft.com/intune/custom-settings-windows-10) to enable the feature. Configure the policy using the following Windows policy configuration service provider (CSP):

Reviewer: ```suggestion
To configure PIN reset on Windows devices you manage, use a Windows Hello for Business configuration in Intune to enable the feature. 
```

> #### Create a PIN Reset Device configuration profile using Microsoft Intune

Reviewer 2: ```suggestion
#### Use an existing WHFB Device configuration profile using Microsoft Intune

If you already have got an existing configuration for Windows Hello for Business, you can enable the feature by enabling the Enable PIN recovery policy in your configuration.

If you do not have an existing configuration, you can create one following the steps below.

#### Create a PIN Reset Device configuration profile using Microsoft Intune
```
The phrasing in the first sentence overuses the word "enable" or "enabling", so I would like a suggestion from @JohanFreelancer9 for a better phrasing.
I am also open to comments on whether this should be a Note blob instead of a separate section.

> 

Reviewer 3: ```suggestion

On an existing configuration for Windows Hello for Business, the feature can be enabled using the Enable PIN recovery policy in your configuration.
 

```

1. Sign-in to [Azure Portal](https://portal.azure.com) using a tenant administrator account.

> 2. Navigate to the Microsoft Endpoint Manager admin center. Click **Devices**. Click **Profiles**. Click **Configuration profiles**. Click **Create profile**.

Reviewer 2: Suggestion:
```suggestion
2. Navigate to the Microsoft Endpoint Manager admin center. Click **Devices > Profiles > Configuration profiles > Create profile**.
```

3. Type **Use PIN Recovery** in the **Name** field. Select **Windows 10 and later** from the **Platform** list.  Select **Identity protection** from the **Profile type** list.
4. Click **Enable** on the "Configure Windows Hello for Business" option. 

> 5. Click **Enable** on the "Enable PIN recovery" option

Reviewer 3: ```suggestion
5. Click **Enable** on the "Enable PIN recovery" option.
```

> 6. Click **OK**

Reviewer 3: ```suggestion
6. Click **OK**.
```

> 7. Click **Create**

Reviewer 3: ```suggestion
7. Click **Create**.
```
